invokeai/frontend/web/src/features/parameters/hooks/useRecallParameters.ts

```diff
@@ -87,8 +88,10 @@ import {
   isValidSteps,
   isValidStrength,
   isValidWidth,
+  isValidVaeModel,
   isValidBoolean,
 } from '../types/parameterSchemas';
+import _ from 'lodash';
```

Comment:
We need to import from `lodash-es` to take advantage of tree-shaking and reduce bundle size

```suggestion
import { isNil } from 'lodash-es';
```